kube-config-files/methode-list-mapper-service.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name:  methode-list-mapper
+  labels:
+    app:  methode-list-mapper
+    visualize: "true"
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: methode-list-mapper
+  template:
+    metadata:
+      labels:
+        app:  methode-list-mapper
+        visualize: "true"
+    spec:
+      containers:
+      - name:  methode-list-mapper
+        image:  up-registry.ft.com/coco/methode-list-mapper:v9
+        imagePullPolicy: Always
+        resources:
+          limits:
+            memory: 512Mi
+        livenessProbe:
+          initialDelaySeconds: 15
+          tcpSocket:
+            port: 8080
+# enable when MLM has a __gtg endpoint
+#        readinessProbe:
+#          httpGet:
+#            path: /__gtg
+#            port: 8080
+#          initialDelaySeconds: 30
+#          periodSeconds: 30
+        env:
+        - name: DOCUMENT_STORE_API_URL
+          value: "document-store-api:8080" 
+        - name: CONCORDANCE_API_URL
+          value: "public-concordances-api:8080"
+        - name: THINGS_API_URL
+          value: "public-things-api:8080"
+        - name: KAFKA_PROXY_URL
+          value: "kafka:8082"
+        ports:
+        - containerPort: 8080
+        - containerPort: 8081
```

Comment:
Had a look and that's the admin endpoint for DW, has metrics and healthchecks there, so I say we should expose it.